packages/plugins/graphql/server/src/bootstrap.ts

```diff
@@ -72,17 +85,14 @@ export async function bootstrap({ strapi }: { strapi: Strapi }) {
     cors: false,
     uploads: false,
     bodyParserConfig: true,
-
-    plugins: [
-      process.env.NODE_ENV === 'production' && !config('playgroundAlways')
-        ? ApolloServerPluginLandingPageDisabled()
-        : ApolloServerPluginLandingPageGraphQLPlayground(),
-    ],
+    // send 400 http status instead of 200 for input validation errors
+    status400ForVariableCoercionErrors: true,
```

Comment:
Wasn't that already the case? Did they change the default behavior in apollo v4?

Reply:
Yeah, it defaults to 200 in apollo v4 but this seems like it would be a pointless breaking change for Strapi to make so we'll just use the old behaviour. I'll make a note of it in the changelog though so nobody is surprised.